jekyll/_cci2/server-3-install-build-services.adoc

```diff
@@ -167,6 +167,66 @@ Provided in the output from `terraform apply`
 * *Nomad Server Certificate Authority (CA) Certificate (required)* - 
 Provided in the output from `terraform apply`
 
+==== Nomad Autoscaler (Optional)
+Nomad provides a utility to automatically scale up or down your nomad clients provided your clients are managed by a cloud provider's autoscaling resource. With Nomad Autoscaler, you only need to provide permission for the utility to manage your autoscaling resource and where it is located. You can enable this resource via KOTS, which will deploy the Nomad Autoscaler service along with your Nomad servers. Below we will go through how to set up Nomad Autoscaler for your provider.
+
+If you do not require this service then feel free to jump to the *Save config* button to update your installation and re-deploy server.
+
+===== AWS
+. Create an IAM user/role and policy for Nomad Autoscaler. You may take one of the following approaches:
+  * Our link:https://github.com/CircleCI-Public/server-terraform/tree/main/nomad-aws[nomad module] creates an IAM user and outputs the keys if you set variable `nomad_auto_scaler = true`. You may reference the example in the link for more details. If you've already created the clients, you can update the variable and run `terraform apply`. The created user's access key and secret will be available in Terraform's output.
```

Comment:
It feels like we should talk about this step earlier so if people know they want to autoscale they can do this at the point of  deploying their nomad clients for the first time.

Also, this feels like something customers want to do after the initial install so I will make a server 3 specific guide to nomad to include this info (up to now we've just been using the nomad guide from v2 - https://circleci.com/docs/2.0/nomad/) - I'll do this in a separate PR once we've merged this into the 3.3 branch though. Let me know if that doesn't sound right.

Reply:
I wasnt sure myself. My thinking though is for first install, they may want to focus on getting things running and can return to autoscaling. its an optional step and does no harm to do this later. 
However I dont have a strong opinion here and am guided by the your experience.

Reply:
I put a note in above to flag this section up. Let's leave as is for now, can always rearrange later.